docs/tutorials/hello_world.rst

```diff
@@ -87,6 +87,8 @@ Update the ``start.php`` to look like this:
     }
     
     function hello_world_page_handler() {
+    	echo elgg_view_resource('hello');
+
     	$params = array(
```

Comment:
You added elgg_view_resource above, but didn't delete the params and elgg_view_page below. Seems odd.